CI_YAML.md

```diff
@@ -54,6 +54,19 @@ targets:
        ["framework", "hostonly"]
 ```
 
+## Adding new targets
+
+All new targets should be added as `bringup: true` to ensure they do not block the tree.
+
+Targets based on the LUCI or Cocoon schedulers will first need to be mirrored to flutter/infra
+before they will be run. This propagation takes about 30 minutes, and will only run as non-blocking
+in postsubmit.
+
+The target will show runs in https://ci.chromium.org/p/flutter (under the repo). See
+https://github.com/flutter/flutter/wiki/Adding-a-new-Test-Shard for up to date information
+on the steps to promote your target to blocking. In general, this involves 10 consecutive green
```

Comment:
For framework, 50 consecutive green runs are needed as of now (based on SLO 2%).

Reply:
Nit: the auto deflake process is running only on framework repo and the 50 is also only applicable to the framework.